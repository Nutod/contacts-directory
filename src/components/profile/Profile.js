import React from "react";
import Sidebar from "../elements/Sidebar/Sidebar";

export default function Profile() {
	return (
		<div style={{ display: "flex" }}>
			<Sidebar />
			<div>Content</div>
		</div>
	);
}
