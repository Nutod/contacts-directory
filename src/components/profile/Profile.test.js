import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import Profile from "./Profile";

Enzyme.configure({ adapter: new Adapter() });

describe("<Profile />", () => {
	const wrapper = shallow(<Profile />);
	it("Should return default true", () => {
		expect(true).toBe(true);
	});

	it("Should match snapshot", () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
