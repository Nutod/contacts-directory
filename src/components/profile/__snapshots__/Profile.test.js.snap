// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> Should match snapshot 1`] = `
<styled.div>
  <Sidebar />
  <styled.div>
    <Styled(styled.div)>
      <styled.p>
        Accounts 
        <span>
           &gt; Profile
        </span>
      </styled.p>
      <styled.div>
        <styled.div>
          <div>
            <p
              style={
                Object {
                  "color": "#ccc",
                  "fontSize": "1rem",
                  "margin": "0",
                }
              }
            >
              Name
            </p>
            <p
              style={
                Object {
                  "margin": "0",
                }
              }
            >
              Larry Etim 
              <styled.span>
                Edit
              </styled.span>
            </p>
          </div>
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "marginTop": "1.5rem",
              }
            }
          >
            <Icons
              label="phone"
            />
            <div
              style={
                Object {
                  "marginLeft": "1rem",
                }
              }
            >
              <p
                style={
                  Object {
                    "color": "#555",
                    "fontSize": "1.3rem",
                    "margin": "0",
                  }
                }
              >
                +234-804-383-3908
                <styled.span>
                  Edit
                </styled.span>
              </p>
              <p
                style={
                  Object {
                    "color": "#ccc",
                    "fontSize": "1rem",
                    "margin": "0",
                    "marginTop": "-0.7rem",
                  }
                }
              >
                Mobile
              </p>
            </div>
          </div>
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "marginTop": "1.5rem",
              }
            }
          >
            <Icons
              label="phone"
            />
            <div
              style={
                Object {
                  "marginLeft": "1rem",
                }
              }
            >
              <p
                style={
                  Object {
                    "color": "#555",
                    "fontSize": "1.3rem",
                    "margin": "0",
                  }
                }
              >
                +234-01-256-290
                <styled.span>
                  Edit
                </styled.span>
              </p>
              <p
                style={
                  Object {
                    "color": "#ccc",
                    "fontSize": "1rem",
                    "margin": "0",
                    "marginTop": "-0.7rem",
                  }
                }
              >
                Other
              </p>
            </div>
          </div>
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "marginTop": "1.5rem",
              }
            }
          >
            <Icons
              label="mail"
            />
            <div
              style={
                Object {
                  "marginLeft": "1rem",
                }
              }
            >
              <p
                style={
                  Object {
                    "color": "#555",
                    "fontSize": "1.3rem",
                    "margin": "0",
                  }
                }
              >
                larryetim@gmail.com
                <styled.span>
                  Edit
                </styled.span>
              </p>
              <p
                style={
                  Object {
                    "color": "#ccc",
                    "fontSize": "1rem",
                    "margin": "0",
                    "marginTop": "-0.7rem",
                  }
                }
              >
                Email
              </p>
            </div>
          </div>
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "marginTop": "1.5rem",
              }
            }
          >
            <Icons
              label="suitcase"
            />
            <div
              style={
                Object {
                  "marginLeft": "1rem",
                }
              }
            >
              <p
                style={
                  Object {
                    "color": "#555",
                    "fontSize": "1.3rem",
                    "margin": "0",
                  }
                }
              >
                Rexxe Powerplant
                <styled.span>
                  Edit
                </styled.span>
              </p>
              <p
                style={
                  Object {
                    "color": "#ccc",
                    "fontSize": "1rem",
                    "margin": "0",
                    "marginTop": "-0.7rem",
                  }
                }
              >
                Word
              </p>
            </div>
          </div>
        </styled.div>
        <Styled(styled.div)>
          <Styled(styled.div)>
            <Styled(styled.img)
              src="user.jpg"
            />
          </Styled(styled.div)>
          <Styled(styled.p)>
            Change Image
          </Styled(styled.p)>
        </Styled(styled.div)>
      </styled.div>
    </Styled(styled.div)>
  </styled.div>
</styled.div>
`;
